nrm_django/contacts/admin.py

```diff
@@ -0,0 +1,10 @@
+from django.contrib import admin
+
+from .models import Contact
+
+
+class ContactAdmin(admin.ModelAdmin):
+    pass
```

Comment:
Just did the bare minimum in admin for now, thinking that when we pick up #156 we'll have a better idea of what should be visible, inline, or just hidden.